onnxruntime/core/platform/env.h

```diff
@@ -108,8 +107,11 @@ class Env {
 
   virtual int GetNumCpuCores() const = 0;
 
-  // This function doesn't support systems with more than 64 logical processors
-  virtual std::vector<size_t> GetThreadAffinityMasks() const = 0;
+  // return default threadpool size, and set affinity vector
```

Comment:
>// return default threadpool size, and set affinity vector [](http://example.com/codeflow?start=2&length=58)

Given two return values, what does the user do if they are different?

Reply:
Now it only represents a buffer of memory pertaining to a particular system. For POSIX, it is what is used to be.
For windows, it is simply a memory buffer saving a "group id - bitmask" pairs.
Note that we are not saving each mask to each size_t since they might be of different size.